scalafix-core/shared/src/main/scala/scalafix/v1/Sym.scala

```diff
@@ -0,0 +1,149 @@
+package scalafix.v1
+
+import scala.meta.Tree
+import scala.meta.internal.{semanticdb3 => s}
+import scala.meta.internal.semanticdb3.SymbolInformation.{Property => p}
+import scala.meta.internal.semanticdb3.Scala._
+
+final class Sym private (val value: String) {
+  def isNone: Boolean = value.isNone
+  def isRootPackage: Boolean = value.isRootPackage
+  def isEmptyPackage: Boolean = value.isEmptyPackage
+  def isGlobal: Boolean = value.isGlobal
+  def isLocal: Boolean = value.isLocal
+  def owner: Sym = Sym(value.owner)
+  def info(doc: SemanticDoc): Sym.Info = doc.info(this)
+
+  override def toString: String =
+    if (isNone) "Sym.None"
+    else value
+  override def equals(obj: Any): Boolean =
+    this.eq(obj.asInstanceOf[AnyRef]) || (obj match {
+      case s: Sym => value == s.value
+      case _ => false
+    })
+  override def hashCode(): Int = value.hashCode
+}
+
+object Sym {
+  val RootPackage: Sym = new Sym(Symbols.RootPackage)
+  val EmptyPackage: Sym = new Sym(Symbols.EmptyPackage)
+  val None: Sym = new Sym(Symbols.None)
+  def apply(sym: String): Sym = {
+    if (sym.isEmpty) Sym.None
+    else {
+      if (!sym.startsWith("local")) {
+        sym.desc // assert that it parses as a symbol
+      }
+      new Sym(sym)
+    }
+  }
+
+  object Root {
```

Comment:
RootPackage?

Reply:
Removed in favor of 
```
case `RootPackage` =>
```

since there is already a `val RootPackage = ...`